generated/nidaqmx/constants.py

```diff
@@ -850,27 +869,28 @@ class ThermocoupleType(Enum):
 
 class TimeUnits(Enum):
     SECONDS = 10364  #: Seconds.
-    TICKS = 10304  #: Timebase ticks.
     FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.
+    TICKS = 10304  #: Timebase ticks.
 
 
 class Timescale(Enum):
+    UTC = 15987  #: .
     USE_HOST = 16126  #: Use the host device.
     USE_IO_DEVICE = 16127  #: Use the I/O device.
 
 
 class TimestampEvent(Enum):
-    START_TRIGGER = 12491  #: Start Trigger timestamp.
-    REFERENCE_TRIGGER = 12490  #: Reference Trigger timestamp.
-    ARM_START_TRIGGER = 14641  #: Arm Start Trigger timestamp.
-    FIRST_SAMPLE = 16130  #: First Sample timestamp.
+    START_TRIGGER = 12491  #: Start Trigger
+    REFERENCE_TRIGGER = 12490  #: Reference Trigger
+    ARM_START_TRIGGER = 14641  #: Arm Start Trigger
+    FIRST_SAMPLE_TIMESTAMP = 16130  #: First Sample Timestamp
```

Comment:
Prefer old since its less repetitive

Reply:
Updated enum values as in expected constants.py